scripts/ansible/roles/ava-install/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+ava_daemon_bin_dir: "/usr/bin"
+ava_daemon_data_dir: "/var/lib/{{ ava_daemon_user }}"
+ava_daemon_db_dir: "{{ ava_daemon_data_dir }}/db"
```

Comment:
@StephenButtolph just recently had defined that the data dir would be `~/.gecko`. But maybe for ansible deployments this is fine here?

In any case, note that due to `ava_daemon_user: ava`, this means that only one user will be able to run `ava` on such a deployment.

Reply:
I would advocate that having the directory `/var/lib/ava/.gecko/{staking,db}` is not what a user would expect`. Hence why I overrode it.

Reply:
It's worth pointing out that `ava_daemon_user: ava` and others are only default values. Ansible is _very_ flexible about overriding variables. For instance one could do

```yaml
- hosts: localhost
  become: true
  roles:
    - {name: ava-user, vars: {ava_daemon_user: snoopy}
    - {name: ava-user, vars: {ava_daemon_user: woodstock}
```